mercury_app/tests.py

```diff
@@ -136,6 +214,31 @@ def test_home(self):
         self.assertEqual(response.status_code, 302)
 
 
+class SelectEventsLoggedTest(TestBase):
+
+    def setUp(self):
+        super(SelectEventsLoggedTest, self).setUp()
```

Comment:
Just FTR, you don't need to add the `class, obj` parameters for `super()` in py3. You could do:
`super().setUp()`